src/main/scala/sbtdocker/DockerBuild.scala

```diff
@@ -113,19 +113,23 @@ object DockerBuild {
     var lines = Seq.empty[String]
 
     def runBuild(buildKitSupport: Boolean): Int = {
+      val buildX = if (buildOptions.platforms.isEmpty) Nil else List("buildx")
+      val load = if (buildOptions.platforms.isEmpty) Nil else List("--load")
```

Comment:
The `--load` parameter is needed for `buildx` to create the image

Reply:
Doesn't the `--load` flag only work for single-platform builds? 
From [docker docs](https://docs.docker.com/engine/reference/commandline/buildx_build/#output):
> Currently, multi-platform images cannot be exported with the docker export type. The most common usecase for multi-platform images is to directly push to a [registry](https://docs.docker.com/engine/reference/commandline/buildx_build/#registry) (see registry).

Reply:
Pushing directly to a registry makes breaking changes in this library. Feel free to create a PR that supports multiple platforms in one build